packages/types/tools/copy-ast-spec.ts

```diff
@@ -60,14 +60,16 @@ async function copyFile(
     encoding: 'utf-8',
   });
 
-  await execAsync('yarn', ['prettier', '--write', outpath], {});
+  await execAsync('yarn', ['prettier', '--write', outpath], {
+    shell: true,
```

Comment:
could we instead add this in the `execAsync` function instead of in both callsites?